modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-test/src/testIntegration/java/com/liferay/headless/commerce/admin/order/resource/v1_0/test/OrderResourceTest.java

```diff
@@ -109,81 +140,95 @@ public void testGetOrder() throws Exception {
 		Assert.assertNotNull(actualOrderItems[0].getShippingAddress());
 	}
 
+	@Ignore
 	@Override
 	@Test
-	public void testGetOrdersPage() throws Exception {
+	public void testGraphQLDeleteOrder() throws Exception {
+		super.testGraphQLDeleteOrder();
 	}
 
 	@Override
 	@Test
-	public void testGetOrdersPageWithFilterDateTimeEquals() throws Exception {
-	}
+	public void testPatchOrder() throws Exception {
+		Order postOrder = orderResource.postOrder(randomOrder());
 
-	@Override
-	@Test
-	public void testGetOrdersPageWithPagination() throws Exception {
-	}
+		Order randomPatchOrder = randomPatchOrder();
 
-	@Override
-	@Test
-	public void testGetOrdersPageWithSortDateTime() throws Exception {
-	}
+		orderResource.patchOrder(postOrder.getId(), randomPatchOrder);
 
-	@Override
-	@Test
-	public void testGetOrdersPageWithSortInteger() throws Exception {
-	}
+		Order expectedOrder = postOrder.clone();
 
-	@Override
-	@Test
-	public void testGraphQLDeleteOrder() throws Exception {
-	}
+		BeanTestUtil.copyProperties(randomPatchOrder, expectedOrder);
 
-	@Override
-	@Test
-	public void testGraphQLGetOrder() throws Exception {
-	}
+		Order getOrder = orderResource.getOrder(postOrder.getId());
```

Comment:
I think here we should apply this rule https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DWhere-there's-a-2%2C-there's-a-1

Reply:
This code is the same that buildREST usually generates. In this case I manually wrote it here because the output of the patch API returns void